docker-compose.yml

```diff
@@ -24,7 +24,6 @@ services:
       TEST_STREAM_NAME: test_rabblerouser_stream
     command: sleep 2147483647 # Sleep forever, we'll shell into the container later
     stdin_open: true
-    user: $UID
```

Comment:
Is there a reason you removed this? IIRC, we need this to make it run properly on a Linux host, as discovered by @akjones.